lms/djangoapps/teacher_dashboard/utils.py

```diff
@@ -0,0 +1,9 @@
+from django.conf import settings
```

Comment:
> We are taking from settings LABSTER_API_AUTH_TOKEN. I don't think that settings are reudndant

Yes, after moving `_send_request` in this file it is important to have it there.